pallets/did/src/mock.rs

```diff
@@ -81,7 +80,8 @@ impl did::Config for Test {
 
 pub type TestDidIdentifier = <Test as did::Config>::DidIdentifier;
 
-pub const DEFAULT_ACCOUNT: AccountId = AccountId::new([0u8; 32]);
+#[allow(dead_code)]
```

Comment:
this is only used in the tests? Then i would use

```suggestion
#[cfg(test)]
```